Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:alpine3.7
```

Comment:
FROM golang as builder

Reply:
Note: it doesn't matter what image you use as the first stage as it gets blown away later.

Reply:
@robbyoconnor I'm very familiar with multistage builds, thanks. :-D

Reply:
There's no reason to pull in all of the golang image in to build this.  The alpine image is less than half the size.  Even it should go on a diet.